ApplicationCode/GeoMech/GeoMechDataModel/RigFemPart.cpp

```diff
@@ -441,3 +553,12 @@ size_t RigFemPart::resultValueIdxFromResultPosType( RigFemResultPosEnum resultPo
     CVF_ASSERT( false );
     return 0u;
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+bool RigFemPart::isHexahedron( size_t elementIdx ) const
```

Comment:
You found the best home for this function, will make the code more robust and readable. 👍